src/app/login.tsx

```diff
@@ -1,18 +1,45 @@
 import { useRouter } from 'expo-router';
+import { showMessage } from 'react-native-flash-message';
 
+import { useSignin } from '@/api/auth/use-singin';
 import { LoginForm } from '@/components/login-form';
 import { useAuth } from '@/core';
 import type { LoginFormProps } from '@/types/auth/auth-types';
-import { FocusAwareStatusBar } from '@/ui';
+import { FocusAwareStatusBar, showErrorMessage } from '@/ui';
 
 export default function Login() {
   const router = useRouter();
   const signIn = useAuth.use.signIn();
 
+  const { mutate: singin, error } = useSignin();
+
   const onSubmit: LoginFormProps['onSubmit'] = (data) => {
-    console.log(data);
-    signIn({ access: 'access-token', refresh: 'refresh-token' });
-    router.push('/(tabs)');
+    singin(
+      {
+        user: {
+          email: data.email,
+          password: data.password,
+        },
+      },
+      {
+        onSuccess: (response) => {
+          showMessage({
+            message: 'succes',
+            type: 'success',
+          });
+
+          const authorizationHeader = response.headers.authorization;
+
+          if (authorizationHeader) {
+            signIn(authorizationHeader.toString());
```

Comment:
To avoid confusion, consider renaming either the `signIn` storage function or the one coming from the mutation.